core/common/src/main/java/alluxio/wire/Configuration.java

```diff
@@ -43,7 +43,8 @@
   private final String mPathConfHash;
 
   /**
-   * @return new configuration builder
+   * @return a new {@link Builder} for cluster- and path-level
```

Comment:
Please. define a summary for the javadoc (no problem if it is a little bit redundant)

Reply:
According to the [new Alluxio convention](https://github.com/Alluxio/alluxio/pull/11797/files), getters and setters should omit the method description if it is redundant and only use `@param` and `@return` descriptions. This is recent.